roles/zuul/files/jobs/dib.yaml

```diff
@@ -0,0 +1,38 @@
+- publisher:
+    name: bonnyci-logs
+    publishers:
+      - scp:
+          site: bonnyci-scp
+          files:
+            - target: 'logs/$LOG_PATH'
+              source: 'logs/**'
+              keep-hierarchy: true
+              copy-after-failure: true
+
+- job-template:
```

Comment:
We're not doing anything different in the two pipelines, so I think we can just create a single job rather than use a job-template to build out multiple jobs.